acceptance/tests/storeconfigs/deactivated_nodes.rb

```diff
@@ -84,11 +82,13 @@ class reactivated {
 
     step "deactivate the exporter node" do
       # Deactivate the node and wait until it's been processed
-      on database, "puppet node deactivate '#{exporter.node_name}'"
+      # FYI `puppet node deactivate` and `puppet node status` rely on the the
+      # pdb terminus, so they can only be run on the master
```

Comment:
@kbarber this tripped me up for while because I was using a separate beaker config that had the database on a different host. I changed the node specification just to be explicit.